packages/core/src/Stores/traders-hub-store.js

```diff
@@ -478,6 +479,12 @@ export default class TradersHubStore extends BaseStore {
         return this.selected_region === 'EU';
     }
 
+    get real_account_needed_for_cashier() {
+        const { has_svg_account, has_maltainvest_account } = this.root_store.client;
+
+        return ((has_svg_account && this.is_eu_user) || (has_maltainvest_account && !this.is_eu_user)) && this.is_real;
+    }
```

Comment:
@thisyahlen-deriv Please create a hook in the `@deriv/hooks` package for new logic instead of adding more logic into stores 🙏🏻